Modules/ThirdParty/MetaIO/src/MetaIO/src/metaUtils.cxx

```diff
@@ -1227,7 +1227,7 @@ bool MET_Read(std::istream &fp,
         MET_FieldRecordType * mF = new MET_FieldRecordType;
         MET_InitReadField(mF, s, MET_STRING, false);
         MET_ASCII_CHAR_TYPE * str = (MET_ASCII_CHAR_TYPE *)(mF->value);
-        fp.getline( str, 500 );
+        fp.getline( str, sizeof(mF->value) );
```

Comment:
This commit should go into https://github.com/Kitware/MetaIO, and then we can update ITK's copy by running the update script.

Reply:
@dzenanz 
> This commit should go into https://github.com/Kitware/MetaIO, and then we can update ITK's copy by running the update script.

OK, here it is: https://github.com/Kitware/MetaIO/pull/93  😃